### PR TITLE
Feature html css

### PR DESCRIPTION
**Pull request by obediobadiah**
In the changes below I do:

- The Cards section and Stylization;
- The Work section and the Stylization.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,6 +40,183 @@ <h1>I'm Obedi<br> Glad to see you!</h1>
             </ul>
         </div>
     </section>
+    <section class="card_section">
```
Comment (micheaol) [inline index.html:43]: Please ensure that the main container of the projects has a `display:grid` according to the project requirement.

File: style.css [hunk 2]
```diff
@@ -75,3 +94,163 @@ header nav {
...
   margin-left: 20px;
 }
+
+.cards_container {
+  background-color: white;
+  border-radius: 20px;
+  border: 2px solid rgb(211, 211, 211);
+  padding: 30px;
+  margin: 200px 30px 30px 30px;
+}
```
Comment (micheaol) [inline style.css]: Please ensure that the content of the project is aligned with the `flexbox` according to the project requirement.

File: index.html [hunk 3]
```diff
@@ -40,6 +40,183 @@ <h1>I'm Obedi<br> Glad to see you!</h1>
...
+                    <p>2015</p>
+                </div>
+                <p class="card_paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="card_lang_flex">
+                    <p><strong>html</strong></p>
+                    <p><strong>css</strong></p>
+                    <p><strong>javascript</strong></p>
+                </div>
```
Comment (micheaol) [inline index.html]: Please ensure that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 4]
```diff
@@ -40,6 +40,183 @@ <h1>I'm Obedi<br> Glad to see you!</h1>
...
+                <div class="card_button">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </div>
```
Comment (micheaol) [inline index.html:78]: Please ensure that you stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
image | image

File: index.html [hunk 5]
```diff
@@ -40,6 +40,183 @@ <h1>I'm Obedi<br> Glad to see you!</h1>
...
+        <div class="about_descript">
+            <p> I’m a software developer! I can help
+                you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you
+                need coded, don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="about_connect_links">
+            <a href="#">LET'S CONNECT</a>
+            <ul>
```
Comment (micheaol) [inline index.html]: Please ensure that you stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
image | image

File: index.html [hunk 6]
```diff
@@ -40,6 +40,183 @@ <h1>I'm Obedi<br> Glad to see you!</h1>
...
+        </div>
+        <div class="about_descript">
+            <p> I’m a software developer! I can help
+                you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you
+                need coded, don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="about_connect_links">
+            <a href="#">LET'S CONNECT</a>
```
Comment (micheaol) [inline index.html]: Please ensure that other titles (e.g., "Let's connect") use a header tag `hx`

File: style.css [hunk 7]
```diff
@@ -75,3 +94,163 @@ header nav {
...
+  border-radius: 10px;
+  margin: 20px 0 20px 0;
+}
+
+.about_section {
+  padding: 200px 30px 200px 30px;
+  background-color: white;
+  border-top-right-radius: 100px;
+  margin-top: 200px;
+}
```
Comment (micheaol) [inline style.css]: Please ensure that the content in this section `(about_section)` is aligned with flexbox according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @obediobadiah , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by mikethreels — APPROVED** [review]
Hi Obedi, 

Your project is complete! it's time to merge it :shipit: 
Congratulations! 🎉 

### Because the below wasn't mentioned by the previous reviewer this is not a requirement, however, I do want to encourage you to make the below changes
- [ ] [OPTIONAL] please make sure to use[ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**